orchagent/aclorch.cpp

```diff
@@ -4583,7 +5307,7 @@ void AclOrch::doAclTableTask(Consumer &consumer)
         }
         else if (op == DEL_COMMAND)
         {
-            if (removeAclTable(table_id))
+            if (removeAclTableWithEgrDscp(table_id))
```

Comment:
Suggest keeping the original function name and add the check into `removeAclTable`. Otherwise people may get confused when to use `removeAclTable` and when to use `removeAclTableWithEgrDscp`

Reply:
the removeAclTable is the existing function. the removeAclTableWithEgrDscp enhances the functionality without touching the orignal well tested code. I think its better to keep them segrigated. this way when someone is making changes to this code they can easily figure out that removeAclTableWithEgrDscp was added as a separate effort.

Reply:
Reopen this comment.
Suggest using the original function name and add the logic inside `removeAclTable` so that other maintainers can be clearer about which function to be used.

Reply:
done